notebooks/NIRCAM/Imaging/requirements.txt

```diff
@@ -1,6 +1,3 @@
-numpy<2.0
-jwst==1.15.1
+jwst==1.17.1
 astroquery
-jdaviz
-astropy>=4.3.1
-gwcs==0.21.0
\ No newline at end of file
+jdaviz
```

Comment:
Does not hurt to also pin min version of jdaviz where this new name was introduced but doesn't matter if only care about latest release always.